### ISSUE TITLE
Log error with SubView in 2022.10

### ISSUE THREAD

**Issue by chertvl** [zanna-37/hass-swipe-navigation#13]
When I try to swipe in SubView dashboard (feature of 2022.10), I got an error in log every time I swipe:

```
...
2022-10-06 13:48:58.921 ERROR (MainThread) [frontend.js.latest.202210050] https://url/hacsfiles/hass-swipe-navigation/swipe-navigation.js?hacstag=501725479161:121:54 Uncaught TypeError: Cannot read properties of null (reading 'querySelector')
2022-10-06 13:49:27.110 ERROR (MainThread) [frontend.js.latest.202210050] https://url/hacsfiles/hass-swipe-navigation/swipe-navigation.js?hacstag=501725479161:121:54 Uncaught TypeError: Cannot read properties of null (reading 'querySelector')
...
```

There is no any possible to swipe in SubView, its all ok, but this error is really anoying ;d  
Can you mute/disable this error?

**Comment by zanna-37** [comment]
Sure, I noticed that as well and it's already fixed in the latest release `v1.6.2`.
The logger still produces the following warning:
`↔️ Swipe navigation: [W] Can't determine the active tab`
But this is expected behavior. But if you disagree, we can discuss that.

Can you confirm that it's all fixed now?

**Comment by chertvl** [comment]
> Can you confirm that it's all fixed now?

Looks like its gone, no any errors now, but I dont see warnings too (maybe because of logger level, but I like it).

Thank you for really quick response and fix!
